bin/index

```diff
@@ -1,14 +1,9 @@
 #!/usr/bin/env python
 
 import sys
-import json
 import logging
 import argparse
 
-try: from urllib import parse as urlparse
-except ImportError: # Python 2.x
-    import urlparse
-
 import requests
 
 from index import errors
```

Comment:
If we want to keep this functionality we should probably fix these imports. It should be `from indexclient` instead of `from index`.

I wonder if anyone has tinkered with this yet. It seems to work. I used `jq` to make the paste less noisy.
```
> ./bin/index --host 'indexd.service.consul' --port 80 retrieve 00000073-27e1-4dcd-bfdc-e458c31feec2 | jq '.did,.created_date'
"00000073-27e1-4dcd-bfdc-e458c31feec2"
"2021-12-14T01:47:28.566542"

```